src/tests/ftest/util/command_utils.py

```diff
@@ -149,17 +159,22 @@ def _run_process(self):
         # Clear any previous run results
         self.result = None
         command = str(self)
+        if raise_exception is not None:
+            do_raise = raise_exception
+        else:
+            do_raise = self.exit_status_exception
```

Comment:
We could save ourselves an extra variable here:
```suggestion
        if raise_exception is None:
            raise_exception = self.exit_status_exception
```